### PR TITLE
Aijar-ISS-9: Add rest end point to check for public holiday

### PR DESCRIPTION
**Pull request by reagan-meant**
A clean PR for [this](https://github.com/METS-Programme/openmrs-module-aijar/pull/495)

### CODE REVIEW

File: api/src/test/resources/org/openmrs/module/aijar/include/standardTestDataset.xml [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 	<global_property  property="ugandaemr.autoCloseVisit.visitTypeUUID"  property_value="7b0f5697-27e3-40c4-8bae-f4049abfb4ed"  description="The UUID of the visit type that will be closed"  uuid="7ac62fa5-4515-4bf6-8f0e-876e8f85d81d"/>
 	
+	<public_holiday public_holiday_id="1" public_holiday_date="2020-01-01" name="New Years Day" description="Start of a new year" creator="1" date_created="2016-04-05 16:43:19.0" voided="0" uuid="8d5b27bc-c2cc-11de-8d13-0010c6dffd0f"/>
+	<public_holiday public_holiday_id="2" public_holiday_date="2020-05-01" name="Labour Day" description="A day for workers to celebrate" creator="1" date_created="2016-04-05 16:43:19.0" voided="0" uuid="8r5b27bc-c2cc-11de-8d13-0010c6dffd0f"/>
+	<public_holiday public_holiday_id="3" public_holiday_date="2020-03-08" name="Womenz Day" description="A Day for women" creator="1" date_created="2016-04-05 16:43:19.0" voided="0" uuid="8r5b27bc-c2cc-13de-8d13-0010c6dffd0f"/>
```
Comment (ebichete) [inline api/src/test/resources/org/openmrs/module/aijar/include/standardTestDataset.xml:8]: "International Women's Day", please. Let's be more discerning with our user-visible strings.

Comment (reagan-meant) [reply api/src/test/resources/org/openmrs/module/aijar/include/standardTestDataset.xml:8]: Thanks @ebichete

**Comment by ssmusoke** [comment]
@reagan-meant Can you add a Liquibase changeset for the public holidays in 2020 and 2021

**Comment by ssmusoke** [comment]
@reagan-meant Do not put the public holidays sample data in a separate SQL file, liquibase does not support that option, put the contents in a single changeset

Also use REPLACE INTO instead of INSERT INTO to handle duplicate inserts

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -306,4 +306,42 @@
...
+            <column name="date_changed" type="datetime"/>
+            <column name="voided" type="TINYINT(3)"/>
+            <column name="date_voided" type="datetime"/>
+            <column name="voided_by" type="INT"/>
+            <column name="void_reason" type="VARCHAR(255)"/>
+            <column name="uuid" type="VARCHAR(38)"/>
+        </createTable>
+    </changeSet>
+	
+	<changeSet id="ugandaemr-17052020-iss-9" author="rmakoba">
```
Comment (ssmusoke) [inline api/src/main/resources/liquibase.xml:342]: @reagan-meant check that the public_holiday table exists, if it does not then continue 

<preConditions onFail="CONTINUE">
</preConditions>

Comment (reagan-meant) [reply api/src/main/resources/liquibase.xml:342]: Done

**Pull request merged**